manager/Makefile

```diff
@@ -52,7 +52,7 @@ run-notebook-tests: export DOCKER_NAMESPACE?=fybrik-system
 run-notebook-tests: export DOCKER_TAGNAME?=master
 run-notebook-tests: export USE_MOCKUP_CONNECTOR?=true
 run-notebook-tests: wait_for_manager
-	cd testdata/notebook && ./setup.sh
+	#cd testdata/notebook && ./setup.sh
```

Comment:
Why commented out? If not needed, remove the comment and delete the setup script.

Reply:
Left over from testing. Thanks